src/test/alert_tests.cpp

```diff
@@ -162,8 +162,8 @@ BOOST_AUTO_TEST_CASE(AlertNotify)
     SetMockTime(11);
     const std::vector<unsigned char>& alertKey = Params(CBaseChainParams::MAIN).AlertKey();
 
-    boost::filesystem::path temp = GetTempPath() / "alertnotify.txt";
-    boost::filesystem::remove(temp);
+    boost::filesystem::path temp = GetTempPath() /
+        boost::filesystem::unique_path("alertnotify-%%%%.txt");
```

Comment:
Meh. Sorry. I oversaw the line break... all good!
You can always use `clang-format -i <file>` if you wan't to confirm to the code-style-guidelines.